### DISCUSSION TITLE
ERSPAN support with latest 2.3.X release?

### DISCUSSION THREAD

**Discussion by RollingMoses** [Security-Onion-Solutions/securityonion#2303] [Unsupported Versions] [answered]
Is there any news around ERSPAN support with the latest release of SO? That may be the only option in one particular environment we have, but older threads from previous versions say it wasn't officially supported. This link from 5 years ago was mentioned in several responses, but I'm hoping there's something a little simpler or more intuitive to configure now?
https://www.google.com/url?q=https%3A%2F%2Fbrezular.com%2F2015%2F05%2F03%2Fdecapsulation-erspan-traffic-with-open-source-tools%2F&sa=D&sntz=1&usg=AFQjCNHr2uo3QiTKA08XP-H4UImBwWDVgg

**Comment by weslambert** [comment, marked as answer]
Both Zeek and Suricata should have ERSPAN support, but we don't currently provide support for decapsulating that traffic without use of external tools like mentioned above.

**Reply by RollingMoses** [reply]
OK. If anyone has experience implementing it on CentOS 7 or the Security Onion ISO in general, I would appreciate any helpful advice/lessons learned. Thanks!
